api/src/champion-mastery/controller.ts

```diff
@@ -0,0 +1,8 @@
+import { API } from 'lambda-api';
```

Comment:
We should use `import type` when we can. In this case, `import type { API } from ...`

Reply:
Will add this in the next revision